crates/lib/src/stdlib/tags/include_tag.rs

```diff
@@ -68,10 +94,25 @@ impl ParseTag for IncludeTag {
 
         let partial = partial.expect_value().into_result()?;
 
-        // no more arguments should be supplied, trying to supply them is an error
-        arguments.expect_nothing()?;
+        let mut vars = Vec::new();
+        while let Ok(next) = arguments.expect_next("") {
+            let id = next.expect_value().into_result()?.to_string();
+
+            arguments
+                .expect_next("expected string")?
+                .expect_str("=")
```

Comment:
Based on the test changes below, it seems this isn't the right syntax but should instead the same syntax used for for-loops.
```rust
fn parse_attr(arguments: &mut TagTokenIter<'_>) -> Result<Expression> {
    arguments
        .expect_next("\":\" expected.")?
        .expect_str(":")
        .into_result_custom_msg("\":\" expected.")?;

    arguments
        .expect_next("Value expected.")?
        .expect_value()
        .into_result()
}

        while let Some(token) = arguments.next() {
            match token.as_str() {
                "limit" => limit = Some(parse_attr(&mut arguments)?),
                "offset" => offset = Some(parse_attr(&mut arguments)?),
                "reversed" => reversed = true,
                _ => {
                    return token
                        .raise_custom_error("\"limit\", \"offset\" or \"reversed\" expected.")
                        .into_err();
                }
            }
        }

```

Reply:
Yep I had assumed incorrectly that jekyll and liquid syntax was going to be the same but found a good syntax guide. So will change it to be the same as the for loop.
liquid: https://www.rubydoc.info/github/Shopify/liquid/Liquid/Include#SYNTAX-constant
jekyll: https://jekyllrb.com/docs/includes/